middle_end/flambda2/to_cmm/to_cmm_shared.ml

```diff
@@ -86,7 +86,7 @@ let memory_chunk_of_kind (kind : Flambda_kind.With_subkind.t) : Cmm.memory_chunk
     ->
     Word_int
   | Naked_number Naked_float -> Double
-  | Naked_number Naked_vec128 -> Onetwentyeight
+  | Naked_number Naked_vec128 -> Onetwentyeight_unaligned
```

Comment:
Please leave a comment as to why "unaligned" is correct here (e.g. not used for bigstrings)